prov/verbs/src/verbs_info.c

```diff
@@ -582,7 +582,9 @@ static int fi_ibv_get_device_attrs(struct ibv_context *ctx, struct fi_info *info
 	info->tx_attr->rma_iov_limit		= device_attr.max_sge;
 
 	info->rx_attr->size 			= MIN(device_attr.max_qp_wr,
-						      device_attr.max_srq_wr);
+						      device_attr.max_srq_wr ?
+						      device_attr.max_srq_wr :
+						      device_attr.max_qp_wr);
```

Comment:
Can we re-order this check to move it outside the MIN call?  It seems odd taking the MIN of the same value.